packages/orchestrator-ui-components/src/components/WFOSteps/WFOStep/WFOStep.tsx

```diff
@@ -37,6 +45,7 @@ export const WFOStep = ({
                 css={{
                     gap: 0,
                     alignItems: 'center',
+                    marginBottom: 20,
```

Comment:
Try to keep styling in a separate styles.ts file. More important, try to use the base value (theme.base) -- see packages/orchestrator-ui-components/src/components/WFOTimeline/styles.ts as example

Reply:
So why is the gap and alignItems ok here?

Reply:
There are at least 6 other occurences of inline CSS. I will resolve them; no problem -> but you might review stuff better before it's merged. Added it to the todo list.

Reply:
Only since recent I came up with a pattern to move the inline css out of the components and having the theme available in that separate file. You will find many more places in the codebase where we still have inline CSS. So my comment about inline you can see as a "you could....". Though I would like to suggest to use the base value from the theme as a base to come up with these values

Reply:
@OFFTOPIC:
I now see a lot of css in getStyles.ts which hard codes color. This will get hard to maintain with a dark theme.
We need to come up with some good rules for colors.